### ISSUE TITLE
support :extra-values for specific target paths

### ISSUE THREAD

**Issue by gambhiro** [m0smith/lein-resource#22]
This is for the case where you want to render the templates with different variables based on the target path, to allow using some view-logic say, for a dev and prod output.

As in #1, but allowing for :extra-values to be passed with each path:

```
  :resource {:resource-paths [["resources"
                               {:target-path "app/dev"
                                :extra-values {:dev true :prod false}}]
                              ["resources"
                               {:target-path "app/prod"
                                :extra-values {:dev false :prod true}}]]}
```

**Comment by m0smith** [comment]
Try pulling the branch  feature/gh-22-support-extra-values-in-resource and installing it and seeing if it does what you want

**Comment by gambhiro** [comment]
Beautiful! It works great, many thanks.

**Comment by m0smith** [comment]
Released in version 16.11.1
